nilearn/masking.py

```diff
@@ -513,6 +513,139 @@ def compute_multi_background_mask(data_imgs, border_size=2, upper_cutoff=0.85,
     return mask
 
 
+def compute_gray_matter_mask(target_img, threshold=.5,
+                             connected=True, opening=2, memory=None,
+                             verbose=0):
+    """ Compute a mask corresponding to the gray matter part of the brain.
+    The gray matter part is calculated through the resampling of MNI152
+    template Gray Matter mask onto the image
+
+    Parameters
+    ----------
+    target_img: Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Images used to compute the mask. 3D and 4D images are accepted.
+        Only the shape and affine of target_img will be used here.
+
+    threshold: float, optional
+        The value under which the MNI template is cut off.
+
+    connected: boolean, optional
+        if connected is True, only the largest connect component is kept.
+
+    opening: bool or int, optional
+        if opening is True, a morphological opening is performed, to keep
+        only large structures. This step is useful to remove parts of
+        the skull that might have been included.
+        If opening is an integer `n`, it is performed via `n` erosions.
+        After estimation of the largest connected constituent, 2`n` closing
+        operations are performed followed by `n` erosions. This corresponds
+        to 1 opening operation of order `n` followed by a closing operator
+        of order `n`.
+
+    memory: instance of joblib.Memory or string
+        Used to cache the function call.
+
+    verbose: int, optional
+
+    Returns
+    -------
+    mask: nibabel.Nifti1Image
+        The brain mask (3D image)
+    """
+    if verbose > 0:
```

Comment:
`verbose` seems binary. If so, it should be a `bool`, with default value `False`. 
And the if statement would become:
`if verbose:`

Currently it is of type `int`, indicating that greater numbers will give more details, which is not the case.

Setting it to `True` seems to only print the task of the function, indicated by function name. I think the utility of that is questionable.

This seems to be the case with most functions in this file.

Reply:
Right, I used the same way as for the other remasking strategies. Maybe it is worth another PR to check for the proper use of  `verbose` ?

Reply:
No: verbose being an integer is a valid value. It's then larger gives more verbosity

Reply:
Larger value for verbose here doe not yield more information. Anything above zero gives the same info.